src/bun.js/module_loader.zig

```diff
@@ -2141,6 +2141,10 @@ pub const ModuleLoader = struct {
                 virtual_source = eval_script;
                 loader = .tsx;
             }
+            if (strings.endsWithComptime(specifier, bun.pathLiteral("/[stdin]"))) {
```

Comment:
☝️ Since `strings.endsWithComtime` could be only either `/[eval]/` or `/[stdin]` we could follow-up to collapse this into the one above with an `or`